### DISCUSSION TITLE
Screen lock button?

### DISCUSSION THREAD

**Discussion by schmadde** [osmandapp/OsmAnd#17534] [Q&A]
I am using OSMAnd+ for quite some time as a navigation app form my phone mounted on my bicycle handlebars. 

This winter and spring when it rained basically every day I had big trouble using it. Rain caused "ghost input" so it was not possible to have the map and navigation cues visible. I could not find any way to lock the screen so I can at least see the map and other information while rain is falling on the phone display. 

Is there a way to enable a screen lock, maybe with a button that appears on the map display?

**Comment by DmitryAlexei** [comment]
Have you tried screen control options?
See https://osmand.net/docs/user/navigation/guidance/voice-navigation#screen-control

**Reply by schmadde** [reply]
As far as I understand this documentation those options do something completely different and unrelated. My problem is that I cannot have the screen on but touch disabled.

**Comment by mikehgentry** [comment]
I've been wondering about this (I have a *very* oversensitive touchscreen on my phone, which makes it literally impossible to use in my pocket without reconfiguration)

I've come across this, which seems to work in my limited testing so far

https://play.google.com/store/apps/details?id=biz.obake.team.touchprotector

**Reply by schmadde** [reply]
This does seem like a useable workaround. A Screen control to lock the screen and some sort of button press or code to unlock again would be much nice. I wonder if nobody else has this problem or why it is not an issue that gets addressed?

**Reply by mikehgentry** [reply]
There used to be a built in function in OsmAnd. I have no idea why it was removed (perhaps it was just considered out of scope, especially since it works on several OSes which may handle it differently?)

**Comment by scaidermern** [comment]
Related to #6770
